workshop/content/60-go/20-create-project/500-deploy.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "cdk deploy"
+weight = 500
++++
+
+Okay, we've got a CloudFormation template. What's next? __Let's deploy it into our account!__
+
+## Bootstrapping an environment
+
+The first time you deploy an AWS CDK app into an environment (account/region),
+you can install a "bootstrap stack". This stack includes resources that
+are used in the toolkit's operation. For example, the stack includes an S3
+bucket that is used to store templates and assets during the deployment process.
+
+You can use the `cdk bootstrap` command to install the bootstrap stack into an
+environment:
+
+```
+cdk bootstrap
+```
+
+Then:
+
+```
+ ⏳  Bootstrapping environment aws://999999999999/us-east-1...
+...
+```
+
+{{% notice info %}} You might see an Access Denied error at this step,
+if the **AWS CLI** has not been [set up correctly](/15-prerequisites/200-account.html) or if the active
+[AWS profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)
+does not have the `cloudformation:CreateChangeSet` permission. {{% /notice %}}
+
+## Let's deploy
+
+Use `cdk deploy` to deploy a CDK app:
+
+```
+cdk deploy
+```
+
+You should see a warning like the following:
```

Comment:
"You will be informed of security-related changes that the CDK is going to perform on your behalf:"